Makefile

```
@@ -8,7 +8,7 @@ MDBX_PATH = "crates/storage/libmdbx-rs/mdbx-sys/libmdbx"
 DB_TOOLS_DIR = "db-tools"
 FULL_DB_TOOLS_DIR := $(shell pwd)/$(DB_TOOLS_DIR)/
 
-BUILD_PATH = "target"
+BUILD_PATH := $(if $(CARGO_TARGET_DIR),$(CARGO_TARGET_DIR),target)
```

a simple 
```suggestion
CARGO_TARGET_DIR ?= target
```
should do, and then replace the `BUILD_PATH` with `CARGO_TARGET_DIR` everywhere below